web/src/router.ts

```diff
@@ -28,6 +28,26 @@ const routes: RouteRecordRaw[] = [
     component: (): Component => import('~/views/ReposOwner.vue'),
     props: true,
   },
+  {
+    path: '/org/:repoOwner',
+    name: 'org-wrapper',
+    component: (): Component => import('~/views/org/OrgWrapper.vue'),
+    props: true,
+    children: [
+      {
+        path: '',
+        name: 'org',
+        redirect: (route) => ({ name: 'repos-owner', params: route.params }),
```

Comment:
Or should we do it the other way around and redirect `/:repoOwner` to `/org/:repoOwner` 🤔